test/test_view/test_bloomberg.py

```diff
@@ -67,34 +67,33 @@ def test_login_post_not_authenticated(self, mock_one_login):
         assert res.status_code == 302, res.status_code
 
     @with_context
-    @patch('pybossa.view.account._sign_in_user', autospec=True)
-    @patch('pybossa.view.account.create_account', autospec=True)
+    @patch('pybossa.view.bloomberg.create_account', autospec=True)
     @patch('pybossa.view.bloomberg.OneLogin_Saml2_Auth', autospec=True)
-    def test_login_create_account_fail(self, mock_one_login, mock_create_account, mock_sign_in):
+    def test_login_create_account_fail(self, mock_one_login, mock_create_account):
         redirect_url = 'http://localhost'
         mock_auth = MagicMock()
         mock_auth.get_errors.return_value = False
         mock_auth.process_response.return_value = None
         mock_auth.is_authenticated = False
         mock_one_login.return_value = mock_auth
-        mock_auth.get_attributes.return_value = {'UUID': [u'1234567'], 'FirstName': [u'test'], 'LastName': [u'test'], 'PVFLevels': [u'PVF_GUTS_3'], 'LoginID': [u'test']}
-        mock_create_account.return_value = None
+        mock_auth.get_attributes.return_value = {'UUID': [u'1234567'], 'FirstName': [u'test'], 'emailAddress': ['test@test.com'], 'LastName': [u'test'], 'PVFLevels': [u'PVF_GUTS_3'], 'LoginID': [u'test']}
+        mock_create_account.side_effect = Exception()
```

Comment:
Should we consider [asserting](https://stackoverflow.com/a/129522) that the exception actually occurred, to distinguish this test from the successful case?

```python
self.assertRaises(Exception, create_account)
```

Or to check the actual message contained within the Exception:

```python
with self.assertRaises(Exception) as context:
  # Call method that will raise an exception.
  res = self.app.post('/bloomberg/login', method='POST", ...)
  # Assert than the exception occurred.
  self.assertTrue('Check the exception message string here' in context.exception)
```